src/perm_uses/fixed_hash.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) DUSK NETWORK. All rights reserved.
+// Licensed under the MPL 2.0 license. See LICENSE file in the project root for details.”
+//! The `pad` module implements the Sponge's padding algorithm
+use crate::sponge::pad::*;
+
+use dusk_plonk::prelude::*;
+use hades252::strategies::*;
+use hades252::WIDTH;
+
+
+/// Takes in one BlsScalar and outputs 2. 
+/// This function is fixed.
+pub fn two_outputs(message: BlsScalar) -> [BlsScalar; 2] {
+    let mut strategy = ScalarStrategy::new();
+
+    // The value used to pad the words is zero.
+    let padder = BlsScalar::zero();
+
+    // The capacity is 
+    let capacity = BlsScalar::one() * BlsScalar::from(2<<64-1) + BlsScalar::one(); 
```

Comment:
This could be a constant.

Reply:
I don't want to encode it as a constant for now, incase we opt for the change in number of inputs and outputs.

Reply:
Is there any reason why not? It will be faster and cleaner. The only reason to have that is if you were doing a generic impl over the ammount of inputs/outputs